src/subscription_manager/plugins.py

```diff
@@ -438,6 +438,26 @@ def __init__(self, clazz, filter_string, installed_products):
         self.installed_products = installed_products
 
 
+class PreUnRegisterConduit(BaseConduit):
```

Comment:
Honestly, I would not bother with adding an additional plugin hook here.  If someone wants it, they'll ask for it.  And so far the response to the plugin framework has been underwhelming: I don't think anyone really uses it.